examples/mixer_demo.rs

```diff
@@ -11,26 +11,33 @@ fn main() {
 
     let args: Vec<_> = env::args().collect();
 
-    if args.len() == 2 {
-        demo(Path::new(&args[1]), Path::new(""));
-    } else if args.len() == 3 {
-        demo(Path::new(&args[1]), Path::new(&args[2]));
+    if args.len() > 2 {
+        let sound_file_path;
```

Comment:
`let sound_file = args.get(2).map(|sound_file| Path::new(sound_file))` doesn't work here? Much easier to read and shorter as well.